states/north_dakota/elections.toml

```diff
@@ -7,3 +7,36 @@ original_date = 2020-06-09
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+# [20201103.registration]
+# North Dakota is the only state without voter registration/
+# sources = ["https://vip.sos.nd.gov/pdfs/Portals/votereg.pdf"]
+
+[20201103.absentee.application]
+# No specific deadline is called out, just within time to receive the ballot
+# 14 day pad to send, receive and send in ballot
+email_by = 2020-10-20
+fax_by = 2020-10-20
+in_person_by = 2020-10-20
+sources = ["https://www.legis.nd.gov/cencode/t16-1c07.pdf#nameddest=16p1-07-05"]
+
+[20201103.absentee]
+received_by = 2020-11-02
```

Comment:
https://vip.sos.nd.gov/pdfs/Portals/Voting-MailBallotAbsentee.pdf says 11-02 is a postmarked date. My guess is that 11-09 is the received by.

Reply:
There is a bunch of confusing language in the rules about if you can't read the postmark or if it is postmarked, but no definitive must be received by X if postmarked by Y, so we'll go with the "by canvassing date" as the received by and and put election day as the the postmarked_by and since there is an in_person_by I'll add that as well. 
https://www.legis.nd.gov/cencode/t16-1c07.pdf#nameddest=16p1-07-09

Reply:
I don't think it can be postmarked on election day:

> Any envelope containing an absent voter's ballot with a postmark or official date stamp on the day of election or thereafter may not be tallied with the ballots timely submitted for the election